ignite/config/plugins/config.go

```diff
@@ -4,7 +4,9 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"strings"
 
+	"golang.org/x/exp/slices"
```

Comment:
Maybe we should consider creating our package instead of relying in the experimental repo.

Reply:
Agree:

Make issue for this?

Reply:
Well they (the go team) put `slices` and `maps` in experiment because of the novelty of generics, but I'm pretty confident it is not going to be broken.

Reply:
Additional info, there's a discussion about adding `exp/slices` in the stdlib for 1.21.
https://github.com/golang/go/issues/57433